AAP-16378 Added ldap-ca.crt for the LDAP certificate selection (#594)
ansible/aap-docs: downstream/modules/platform/proc-configuring-controller-ldap-security.adoc

Use this procedure to configure LDAP security for your {ControllerName}.

.Procedure
. If you do not have a `ldap_cacert_secret`, you can create one with the following command:
+
----
$ oc create secret generic <resourcename>-custom-certs \
    --from-file=ldap-ca.crt=<PATH/TO/YOUR/CA/PEM/FILE>  \ <1>
----

> <1> Modify this to point to where your PEM and key files are stored.

Reviewer: ```suggestion
<1> Modify this to point to where your CA cert is stored.
```

Use this procedure to configure LDAP security for your {ControllerName}.

.Procedure

> . Under *LDAP Certificate Authority Trust Bundle* click the drop-down menu and select *ldap-ca.crt*.

Reviewer: The issue reporter is referring to the format of the information inside the secret itself.  This docs section in the upstream docs describes it.

* https://ansible.readthedocs.io/projects/awx-operator/en/latest/user-guide/advanced-configuration/trusting-a-custom-certificate-authority.html

Something like this note is what they are expecting I think:

> Please note the awx-operator will look for the data field ldap-ca.crt in the specified secret when using the ldap_cacert_secret.

It probably would be good to add an example of how to create the secret with the `oc` create secret` command too.

Author: I've updated the doc based on the feedback you provided.

Author: Change implemented. Thank you!
